applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAX.java

```diff
@@ -177,7 +187,7 @@ private void initialize(final String pluginVersion, final String sdkKey, final R
 
         // If SDK key passed in is empty, check Android Manifest
         String sdkKeyToUse = sdkKey;
-        if ( TextUtils.isEmpty( sdkKey ) )
+        if ( !AppLovinSdkUtils.isValidString( sdkKey ) )
```

Comment:
We should still use `TextUtils.isEmpty()` here. We'd always want to use positive case, since it makes it easier to read. 
```suggestion
        if ( TextUtils.isEmpty( sdkKey ) )
```